src/marshmallow/fields.py

```diff
@@ -562,33 +562,33 @@ def _nested_normalized_option(self, option_name: str) -> typing.List[str]:
             if field.startswith(nested_field)
         ]
 
-    def _serialize(self, nested_obj, attr, obj, many=False, **kwargs):
+    def _serialize(self, nested_obj, attr, obj, **kwargs):
         # Load up the schema first. This allows a RegistryError to be raised
         # if an invalid schema name was passed
         schema = self.schema
         if nested_obj is None:
             return None
-        many = schema.many or self.many or many
-        return schema.dump(nested_obj, many=self.many or many)
+        many = schema.many or self.many
+        try:
+            return schema.dump(nested_obj, many=many)
+        except ValidationError as exc:
+            raise ValidationError(exc.messages, valid_data=exc.valid_data) from exc
```

Comment:
I don't understand the need for this try/except block.

Reply:
I don't remember having to manually resolve a merge conflict for this line. It wasn't in the original diff, but it was part of the upstream merge that was performed to resolve merge conflicts. I only reverted the 2 commits I added, but it looks like git reverted b922a7c063 also.

I will fix this.